package.json

```diff
@@ -54,7 +53,7 @@
     "publish:css": "node ./scripts/publish-process/transform-js-scss-css-imports.js && node ./scripts/publish-process/transform-bpk-imports.js && node ./scripts/publish-process/publish-css-tagged-packages.js && git reset --hard HEAD",
     "postpublish": "./scripts/publish-process/update-docs-deps.sh && node ./scripts/publish-process/update-changelog.js",
     "release": "npm run publish",
-    "sassdoc": "sassdoc {packages/bpk-mixins/src/**/*,node_modules/@skyscanner/bpk-svgs/dist/*,node_modules/@skyscanner/bpk-foundations-web/tokens/base.default}.scss -d dist-sassdoc -v --strict",
+    "sassdoc": "sassdoc {node_modules/bpk-mixins/src/**/*,node_modules/@skyscanner/bpk-svgs/dist/*,node_modules/@skyscanner/bpk-foundations-web/tokens/base.default}.scss -d dist-sassdoc -v --strict",
```

Comment:
Does this need to be here? This looks like a responsibility of the project that hosts those files.

Reply:
Agree, good shout!